app/src/main/java/app/revanced/manager/data/platform/Filesystem.kt

```diff
@@ -16,7 +16,7 @@ class Filesystem(private val app: Application) {
      * A directory that gets cleared when the app restarts.
      * Do not store paths to this directory in a parcel.
      */
-    val tempDir = app.cacheDir.resolve("ephemeral").apply {
+    val tempDir = app.filesDir.resolve("ephemeral").apply {
```

Comment:
Use getDir instead

Reply:
Since getDir creates the directory when it is called, I thought it was not suitable because I would have to create the directory myself.
I noticed that getDir and mkdirs() have different permissions on the folders created.
Should I still use getDir?

Reply:
Yes, because that is what the android docs recommends.